client/profile-wizard/steps/store-details.js

```diff
@@ -167,67 +177,110 @@ class StoreDetails extends Component {
 		return (
 			<Fragment>
 				<H className="woocommerce-profile-wizard__header-title">
-					{ __( 'Where is your store based?', 'woocommerce-admin' ) }
+					{ __( 'Welcome to WooCommerce', 'woocommerce-admin' ) }
 				</H>
 				<H className="woocommerce-profile-wizard__header-subtitle">
 					{ __(
-						'This will help us configure your store and get you started quickly',
+						"Tell us about your store and we'll get you set up in no time",
 						'woocommerce-admin'
 					) }
-				</H>
 
-				<Card>
-					<Form
-						initialValues={ this.initialValues }
-						onSubmitCallback={ this.onSubmit }
-						validate={ validateStoreAddress }
+					<Tooltip
+						text={ __(
+							'Your store address will help us configure currency\n options and shipping rules automatically.\n This information will not be publicly visible and can\n easily be changed later.',
+							'woocommerce-admin'
+						) }
 					>
-						{ ( {
-							getInputProps,
-							handleSubmit,
-							values,
-							isValidForm,
-							setValue,
-						} ) => (
-							<Fragment>
-								{ showUsageModal && (
-									<UsageModal
-										onContinue={ () =>
-											this.onContinue( values )
-										}
-										onClose={ () =>
-											this.setState( {
-												showUsageModal: false,
-											} )
-										}
-									/>
-								) }
+						<span className="woocommerce-profile-wizard__tooltip-icon">
+							<Icon icon="info-outline" size={ 16 } />
+						</span>
+					</Tooltip>
+				</H>
+
+				<Form
+					initialValues={ this.initialValues }
+					onSubmitCallback={ this.onSubmit }
+					validate={ validateStoreAddress }
+				>
+					{ ( {
+						getInputProps,
+						handleSubmit,
+						values,
+						isValidForm,
+						setValue,
+					} ) => (
+						<Card>
+							{ showUsageModal && (
+								<UsageModal
+									onContinue={ () =>
+										this.onContinue( values )
+									}
+									onClose={ () =>
+										this.setState( {
+											showUsageModal: false,
+										} )
+									}
+								/>
+							) }
+							<CardBody>
 								<StoreAddress
 									getInputProps={ getInputProps }
 									setValue={ setValue }
 								/>
+							</CardBody>
 
-								<div className="woocommerce-profile-wizard__client">
-									<CheckboxControl
-										label={ __(
-											"I'm setting up a store for a client",
-											'woocommerce-admin'
-										) }
-										{ ...getInputProps( 'isClient' ) }
-									/>
-								</div>
-
-								<Button
-									isPrimary
-									onClick={ handleSubmit }
-									disabled={ ! isValidForm }
-								>
-									{ __( 'Continue', 'woocommerce-admin' ) }
-								</Button>
-							</Fragment>
-						) }
-					</Form>
-				</Card>
+							<CardFooter>
```

Comment:
Making use of the new Card, CardFooter etc from `@wordpress/components`.

I've used 2 footers here to match the design